bitbots_ros_control/src/pressure_converter.cpp

```diff
@@ -160,7 +160,7 @@ void PressureConverter::resetZeroAndScaleValues() {
 
 void PressureConverter::collectMessages() {
   save_zero_and_scale_values_ = true;
-  while (zero_and_scale_values_[0].size() < scale_and_zero_average_) {
+  while (zero_and_scale_values_[0].size() < size_t(scale_and_zero_average_)) {
```

Comment:
What if `scale_and_zero_average_` is negative? The cast would make it a very large integer and the comparison would not make sense. Please cast the left hand side to whatever `scale_and_zero_average_` is (probably `int`).